.github/workflows/sandbox2_build_and_deploy.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 env: 
   VIRTUAL_HOST: sandbox2-videoroom.membrane.work
-  INTEGRATED_TURN_IP: 95.217.153.160
+  EXTERNAL_IP: 95.217.153.160
```

Comment:
It should be changed in every github workflow starting Videoroom, not only in `sandbox` and `sandbox2`